websites/G/Google Maps/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Tofuine",
+		"id": "993496501886136371"
+	},
+	"service": "Google Maps",
+	"description": {
+		"en": "Google Maps is a web mapping platform and consumer application offered by Google. It offers satellite imagery, aerial photography, street maps, 360° interactive panoramic views of streets, real-time traffic conditions, and route planning for traveling by foot, car, bike, air and public transportation."
+	},
+	"url": "www.google.com",
```

Comment:
Would this not interfere with the other Google presences? Sure you can just use `maps.google.com` instead

Reply:
Lol, it does that for me sometimes and on some devices not everything xD